src/server/scripts/Northrend/Ulduar/Ulduar/boss_mimiron.cpp

```diff
@@ -801,7 +801,7 @@ class boss_mimiron : public CreatureScript
                             pInstance->DoUpdateAchievementCriteria(ACHIEVEMENT_CRITERIA_TYPE_KILL_CREATURE, NPC_LEVIATHAN_MKII, 1, me);
 
                         if (hardmode)
-                            if( Creature* computer = me->SummonCreature(NPC_COMPUTER, 2790.0f, 2569.44f, 364.31f, 0.0f, TEMPSUMMON_TIMED_DESPAWN, 1000) )
+                            if( Creature* computer = me->SummonCreature(NPC_COMPUTER, 2746.7f, 2569.44f, 410.39f, 0.0f, TEMPSUMMON_TIMED_DESPAWN, 1000) )
```

Comment:
Same

Reply:
Er... I deleted the repo, so hopefully @FALL1N1 can clean that styling up in their rework of Mimiron.

Reply:
I'm rewriting Mimiron in general, I'll use TC as base, because theirs is blizzlike. Same code but... better...